examples/pipelines/ccfraud/config.yaml

```diff
@@ -63,3 +63,12 @@ training_parameters:
   # file for the CCFRAUD component. Options include: 
   # SimpleLinear, SimpleLSTM, SimpleVAE
   model_name: SimpleLinear
+
+  # Differential privacy
+  dp: true # Flag to enable/disable differential privacy
+  dp_target_epsilon: 50.0 # Smaller epsilon means more privacy, more noise
```

Comment:
Maybe add suggested ranges for epsilon to the comment?

Reply:
Added more info